app/src/main/java/com/arctouch/codechallenge/network/TmdbApi.java

```diff
@@ -26,7 +22,7 @@
     Observable<UpcomingMoviesResponse> upcomingMovies(
             @Query("api_key") String apiKey,
             @Query("language") String language,
-            @Query("page") Long page,
+            @Query("page") int page,
```

Comment:
Why did you make this change?

Reply:
It was to be compliant with the Paging API, But now that you mentioned it would be better to do this cast in the method. It would have been a great impact if other modules of this app were using this method.